src/cargo/ops/cargo_clean.rs

```diff
@@ -23,18 +23,22 @@ pub struct CleanOptions<'cfg> {
     pub requested_profile: InternedString,
     /// Whether to just clean the doc directory
     pub doc: bool,
+    /// If set, doesn't delete anything.
+    pub dry_run: bool,
```

Comment:
Generally our commands with a `--dry-run` have a `shell().warn("aborting add due to dry run")?;`

Reply:
My intent was to have the "Summary" line communicate that it didn't actually delete anything (instead of saying "Removed"). I would prefer to avoid the term "aborting" since it isn't actually aborting anything. Would adding something like `(no files deleted due to --dry-run)` to the summary work?

Reply:
For myself, I'd like us to have some level of consistency between the dry run messages.  For add/remove, I followed the pattern of publish of having a `shell.warn` and for it to follow the same pattern of listing out the operation that was not performed.

Reply:
Ok, I like having a warning. I'm still a little reluctant to use the word "aborting" since it isn't really aborting anything, and seems a little confusing to me, since it is still going through the motions. I went with the wording `warning: no files deleted due to --dry-run`. Does that sound ok?